test-data/unit/check-generic-alias.test

```diff
@@ -0,0 +1,49 @@
+-- Test cases for generic aliases
+
+[case testGenericAliasImportCollections]
+# flags: --python-version 3.9
+from collections import defaultdict, OrderedDict, ChainMap, Counter, deque
+
+t1: defaultdict[int, int]
+t2: OrderedDict[int, int]
+t3: ChainMap[int, int]
+t4: Counter[int]
+t5: deque[int]
+
+[builtins fixtures/tuple.pyi]
+
+[case testGenericAliasImportCollectionsABC]
+# flags: --python-version 3.9
+from collections.abc import Awaitable, Coroutine, Iterable, Iterator, Mapping
+
+t1: Awaitable
+t2: Coroutine
+t3: Iterable
+t4: Iterator
+t5: Mapping
+
+[builtins fixtures/tuple.pyi]
+
+[case testGenericAliasImportBuiltIns]
+# flags: --python-version 3.9
+
+t1: type[int]
+
```

Comment:
This should be enough to add `tuple` to the tests. And for good measure I've added `dict` as well. However `set` needs to be in its own test, as it is now. It seems that the fixtures don't work together for whatever reason.
```suggestion
t2: tuple[int, int]
t3: dict[int, str]

```